### ISSUE TITLE
[1.12.2 v1.8.3] Prism growth lens linking still showing after unlink

### ISSUE THREAD

**Issue by VallenFrostweaver** [HellFirePvP/AstralSorcery#422]
See images below. First image is a linked crystal to a growth lens on a prism over a small wheat farm. Second image is the same crystal unlinked from the prism. The green rays are still there though in all other ways the link is no longer (ie the wheat grows at normal vanilla speed). Been in here since the lens was there but I kept forgetting to report it.
![link](https://user-images.githubusercontent.com/13642054/33711145-dbe01cb4-db3a-11e7-83db-71177159f9b7.png)
![unlink](https://user-images.githubusercontent.com/13642054/33711148-ded9974c-db3a-11e7-89fb-8a02f0d76112.png)

**Comment by HellFirePvP** [comment]
Ah. right. i remember why this exists: Checking if starlight flows through a lens or not is not something i can determine on clientside as from a code-perspective, there is nothing between source (a collector crystal) and the end-receiver. Everything between is essentially just a "loss multiplier" - now this normally isn't a problem since i cache the connections when they are calculated so i can display the beams properly on clientside. Now those colored-lens beams are functioning loosely the same way, but thanks to certain colors of lenses, i can't determine the target as well as i can with lenses that hardly ever change their target (such as damage or regeneration lens colors). the main problem arises that i have to render them somewhat different as i can't batch them. Now the real problem is that thanks to that, i also can't determine if starlight is being transmitted through a lens/prism or not. Normally that isn't a real issue, however it shows with lenses like this...

tl;dr: known problem, fixing it is gonna be weird...
